README.rst

```diff
@@ -1,130 +1,165 @@
-You might also want to visit `our website <http://coala-analyzer.org/>`_.
+.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+
+coala: Language Independent Code Analysis
+-------------
+**coala provides a common command-line interface for linting and fixing all
+your code, regardless of the programming languages you use.**
+
+With coala, users can create rules and standards to be followed in the source
+code. coala has an **user-friendly interface** that is completely customizable.
+It can be used in any environment and is completely modularized.
```

Comment:
im not very sure if the correct word here is ``modular`` or ``modularized`` :-??